samza-core/src/main/java/org/apache/samza/runtime/LocalApplicationRunner.java

```diff
@@ -150,6 +163,10 @@ static MetadataStoreFactory getDefaultCoordinatorStreamStoreFactory(JobConfig jo
   }
 
   private static Optional<CoordinationUtils> getCoordinationUtils(Config config) {
+    if (new JobConfig(config).getConfigLoaderFactory().isPresent()) {
+      config = ConfigUtil.loadConfig(config);
+    }
```

Comment:
Same as above about multiple calls to `loadConfig`.

Reply:
Updated